include/verilated_vcd_c.cpp

```diff
@@ -45,6 +47,19 @@
 # define O_CLOEXEC 0
 #endif
 
+// This size comes form VCD allowing use of printable ASCII characters between
+// '!' and '~' inclusive, which are a total of 94 different values. Encoding a
+// 32 bit code hence needs a maximum of ceil(log94(2**32-1)) == 5 bytes.
+#define VL_TRACE_MAX_VCD_CODE_SIZE 5  ///< Maximum length of a VCD string code
+// We use 8 bytes per code in a suffix buffer array.
+// 1 byte optional separator + VL_TRACE_MAX_VCD_CODE_SIZE bytes for code
+// + 1 byte '\n' + 1 byte suffix size. This luckily comes out to a power of 2,
+// meaning the array can be aligned such that entries never straddle multiple
+// cache-lines.
+#define VL_TRACE_SUFFIX_ENTRY_SIZE 8  ///< Size of a suffix entry
+
+// clang-format on
```

Comment:
Please move this up to just under the #endif.

Please merge master, I fixed clang-format to allow loops on same line which should be closer to what the files were pre clang-format. Then reformat whatever you did previously (what I usually do is make sure git is clean, then reformat, then look at git-diff to see if sane).